engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/execution/JobExecutionEngine.scala

```diff
@@ -152,15 +152,15 @@ final class JobExecutionEngine(
         val remaining = runningJobsRef.updateAndGet(_.filterNot(_.id == jobId))
         logger.log(
           Level.FINEST,
-          "Number of remaining pending jobs: {}",
```

Comment:
Spurious changes

Reply:
The `{}` pattern does not expand when running tests

Reply:
The `{}` pattern is used by SL4J, however this is `java.util.logging` logger. One has to use `{0}` when using `java.util.logging`. Not only in tests, but all the time. The fix is correct.